packages/customize-widgets/src/components/more-menu/style.scss

```diff
@@ -1,35 +0,0 @@
-.customize-widgets-more-menu {
-	margin-left: -4px;
-
-	// the padding and margin of the more menu is intentionally non-standard
-	.components-button {
-		width: auto;
-		padding: 0 2px;
-	}
-
-	@include break-small() {
-		margin-left: 0;
-
-		.components-button {
-			padding: 0 4px;
-		}
-	}
-}
-
-.customize-widgets-more-menu__content .components-popover__content {
-	min-width: 280px;
-
-	// Let the menu scale to fit items.
-	@include break-mobile() {
-		width: auto;
-		max-width: $break-mobile;
-	}
-
-	.components-dropdown-menu__menu {
-		padding: 0;
-	}
-}
-
-.components-popover.customize-widgets-more-menu__content {
-	z-index: z-index(".components-popover.customize-widgets-more-menu__content");
-}
```

Comment:
I don't understand 😅 Why is all of this CSS no longer needed?

Reply:
Aha, the description of https://github.com/WordPress/gutenberg/pull/39113 answered my question!

Reply:
It hasn't really been needed for a while. It's identical to the css implemented by the base component in the interface package:
https://github.com/WordPress/gutenberg/blob/trunk/packages/interface/src/components/more-menu-dropdown/style.scss

I realised I could've deleted it for edit-widgets too, so I've made a small PR here to do that - https://github.com/WordPress/gutenberg/pull/39113

Reply:
Thanks for cleaning it up!